test/unit/mock/profiles/dependencies/inheritance/inspec.yml

```diff
@@ -10,5 +10,3 @@ depends:
     path: ../profile_a
   - name: profile_b
     path: ../profile_b
-  - name: ssh-hardening
```

Comment:
@chris-rock Mostly because the fetching wasn't the main thing we were working so the ssh output was really just cluttering up what we wanted to see.